packages/app-mobile/components/screens/ConfigScreen/SectionSelector.tsx

```diff
@@ -82,8 +82,15 @@ const SectionSelector: FunctionComponent<Props> = props => {
 		}
 	}, [props.selectedSectionName, flatListRef, sections]);
 
+	const containerStyle: ViewStyle = useMemo(() => ({
+		width: props.width,
+		maxWidth: props.width,
+		minWidth: props.width,
```

Comment:
These extra constraints seem necessary to prevent the container from growing/shrinking horizontally (which breaks the sidebar layout).